### PR TITLE
Add support for Pimcore

### PR DESCRIPTION
**Pull request by maff**
This adds support for Pimcore (https://www.pimcore.org/). As we use apache as standard dev setup and ship a `.htaccess` it was more straightforward to create an apache configuration.

### CODE REVIEW

File: scripts/serve-pimcore5.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# install apache and bzip PHP extension
+export DEBIAN_FRONTEND=noninteractive
```
Comment (svpernova09) [inline scripts/serve-pimcore5.sh]: I don't like the idea of having the "install apache" code now in 2 places. That's not necessarily a blocker for getting this PR merged, just something I want to think about a better solution for

**Comment by maff** [comment]
Thanks for the review! I updated the type to `pimcore`.

File: scripts/serve-pimcore5.sh (hunk 1 above)
Comment (maff) [inline scripts/serve-pimcore5.sh]: I see your point - maybe make the apache installation an extra script which can be required from types building on apache?

**Comment by svpernova09** [comment]
Thanks for the PR!

**Pull request merged**